scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -12,7 +12,7 @@
 logger.setLevel(logging.DEBUG)
 
 # pylint: disable=anomalous-backslash-in-string
-result_re = re.compile(".*(PASS|FAIL|SKIP) - (test_)?(.*) in (\d*[.,]?\d*) seconds")
+result_re = re.compile(".*(PASS|FAIL|SKIP) - (test_)?(.*) in (\\d*[.,]?\\d*) seconds")
```

Comment:
I don't see this variable to be used anywhere in this PR, why the change?

Reply:
My IDE was complaining that this was incorrect. I'm not sure what this regex was used for, but for \d to work it definitely needed the extra '\' so this is just my own personal guidelines of "leave the code cleaner than you found it" I'm happy to revert if you'd like me to.

Reply:
I see. I am fine with such explanation. The CI passes, so I believe it is not breaking its performance